Product/Production/Services/DocumentRetrieveCore/src/test/java/gov/hhs/fha/nhinc/docretrieve/nhin/InboundDocRetrieveStrategyImplTest.java

```diff
@@ -57,15 +62,27 @@
     private PolicyTransformer pt;
     private AuditTransformer at;
     private InboundDelegate d;
-    private AuditRepositoryDocumentRetrieveLogger auditLog;
+    private DocRetrieveAuditLogger auditLog;
+       NhinTargetSystemType nhinTargetSystemType;
```

Comment:
This is a bit of a nitpick, but I'm still going to throw it out there :)  The spacing in this entire class is wonky; double/triple spaces, indentation not in intervals of 4 spaces, etc.

Can you do a sanity check on this class, manually if needed, and line things up?